flocker/node/functional/test_docker.py

```diff
@@ -367,6 +367,24 @@ def added(_):
         d.addCallback(added)
         return d
 
+    def test_null_environment(self):
+        """
+        A container that does not include any environment variables contains
+        an empty ``environment`` in the return ``Unit``.
+        """
+        image_name = u"clusterhq/buildroot:latest"
```

Comment:
The last review comment from @exarkun was to use an image hosted on Docker Hub, see previous discussion on this PR.